tests/DCPS/Compiler/idl_test1_main/main.cpp

```diff
@@ -243,20 +269,20 @@ int ACE_TMAIN(int argc, ACE_TCHAR *argv[])
 
   if (!OpenDDS::DCPS::DDSTraits<Xyz::AStruct>::gen_has_key()) {
     ACE_ERROR((LM_ERROR,
-      ACE_TEXT("_dcps_has_key(Xyz::AStruct) returned false when expecting true.\n")
+      ACE_TEXT("gen_has_key(Xyz::AStruct) returned false when expecting true.\n")
       ));
   }
 
   if (OpenDDS::DCPS::DDSTraits<Xyz::StructContainingArrayOfAStructSeq>::gen_has_key()) {
     ACE_ERROR((LM_ERROR,
-      ACE_TEXT("_dcps_has_key(Xyz::StructContainingArrayOfAStructSeq) returned true when expecting false.\n")
+      ACE_TEXT("gen_has_key(Xyz::StructContainingArrayOfAStructSeq) returned true when expecting false.\n")
       ));
   }
 
   Xyz::Foo my_foo;
 
   // greenx < redx < bluex
-  const Xyz::ColorX orig_color = Xyz::greenx;
+  const Xyz::ColorX orig_color = Xyz::ColorX::greenx;
```

Comment:
```suggestion
  const Xyz::ColorX orig_color = Xyz::greenx;
```

Reply:
Yea I had made changes to the C++11 first and by the time I was done I reckoned I made more changes than there was difference between the C++11 and classic `main.cpp`, so I just copied it. Then I removed the parentheses (I still don't understand the need for those) and tweaked it until it would compile.